devices/webcam/setup.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+from setuptools import find_packages, setup
+
+setup(
+    name = "hand_tracking_utility",
+    version = "1.0.0",
+    description = "Example hand tracking estimation visualizaion using LabGraph and Mediapipe",
+    packages = find_packages(),
+    python_requires = ">=3.6",
+    install_requires = [
+        "labgraph==2.0.0",
+        "opencv-python>=4.6.0",
+        "mediapipe>=0.8.11"])
```

Comment:
meidapipe 0.8.11 only supports python 3.7+; I would recommend supporting only python 3.8+ for now as python3.8 end is in 2024 as compared to 2023 for python 3.7.
Reference:
https://pypi.org/project/mediapipe/0.8.11/#files
https://devguide.python.org/versions/

Reply:
Python >=3.8 is now required in setup.py.